docs/connectors/mysql.asciidoc

```diff
@@ -183,6 +183,11 @@ When the connector restarts after having crashed or been stopped gracefully, the
 
 This database history topic is for connector use only, but the connector can optionally generate _schema change events_ on a different topic that is intended for consumer applications. We'll cover this in the link:#schema-change-topic[Schema Change Topic] section.
 
+[NOTE]
+====
+When MySQL connector monitors a table to which a schema change tool like *Gh-ost* or *pt-online-schema-change* is applied then helper tables created during migration process need to be included among whitelisted tables.
```

Comment:
Probably worth adding a second sentence on discarding any events from "ghost" tables via an SMT as per the latest discussion on the ML?

Reply:
@gunnarmorling Added